test/visualTest.ts

```diff
@@ -174,12 +174,12 @@ describe("AsterPlot", () => {
                 expect(linesAfterResize.length).toBeLessThanOrEqual(numOfLabels);
                 expect(labelsAfterResize.length).toBeLessThanOrEqual(numOfLabels);
 
-                const firstLabelX: string = labels[0].getAttribute("x"),
-                    firstLabelY: string = labels[0].getAttribute("y"),
-                    lastLabelY: string = labels[labels.length - 1].getAttribute("y"),
-                    firstResizeLabelX: string = labelsAfterResize[0].getAttribute("x"),
-                    firstResizeLabelY: string = labelsAfterResize[0].getAttribute("y"),
-                    lastResizeLabelY: string = labelsAfterResize[labelsAfterResize.length - 1].getAttribute("y");
+                const firstLabelX: string = labels[0].getAttribute("x")!,
+                    firstLabelY: string = labels[0].getAttribute("y")!,
```

Comment:
It is a better practice to clearly define each variable, such code looks much more clear:
```
const firstLabelX....,
const firstLabelY....,
const firstResizeLabelX....,
```

Reply:
Done.